compiler/src/iree/compiler/Codegen/Common/test/reconcile_translation_info.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-codegen-reconcile-translation-info, canonicalize, cse)))" %s --verify-diagnostics --allow-unregistered-dialect | FileCheck %s
+// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-codegen-reconcile-translation-info{distribute-along=z}, canonicalize, cse)))" %s --verify-diagnostics --allow-unregistered-dialect | FileCheck %s
```

Comment:
I am sorry, I know you are trying to avoid the pain of changing lit tests, but if we are making x the default, then this file should change accordingly. Otherwise it leads to a testing gap cause the default path wont be as tested as a non-default path.